### PR TITLE
[CARBONDATA-761] remove shutdown dictionary server

### PR DESCRIPTION
**Pull request by lionelcao**
remove shutdown dictionary server

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1170/

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala [hunk 1]
```diff
@@ -599,11 +599,6 @@ case class LoadTable(
         // Once the data load is successful delete the unwanted partition files
         try {
 
-          // shutdown dictionary server thread
-          if (carbonLoadModel.getUseOnePass) {
```
Comment (jackylk) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: If this is not shut down, the next load operate will still try to start another dictionary server. 
Do you want to reuse this dictionary server in next load?

**Comment by ravipesala** [comment]
@lionelcao There is another PR https://github.com/apache/incubator-carbondata/pull/656 working on the same cause.

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/91/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/771/

**Pull request closed without merging**